syft/frameworks/torch/tensors/interpreters/large_precision.py

```diff
@@ -106,6 +108,13 @@ def get_class_attributes(self):
 
     @overloaded.method
     def add(self, self_, other):
+        # At this point we can have LPT>AST
+        # If that's the case, delegate into AST to do the operation.
+        # TODO I don't like the idea of this dependency here. An LPT should not know about AST
+        if hasattr(self_, "child") and isinstance(self_.child, AdditiveSharingTensor):
+            self_ = self_.child
+        if hasattr(other, "child") and isinstance(other.child, AdditiveSharingTensor):
+            other = other.child
```

Comment:
I don't think you need these line, if @ overloaded.method was done correctly then self_ and other are already the lpt child so they are AST

Reply:
Without them, this runs fine on my side
```python
lpt = torch.Tensor([-2., 4]).fix_prec(precision_fractional=100).share(alice, bob, crypto_provider=dan)
lpt + lpt
```

Reply:
It was previous to have the backward/forward funcs working :)
Removed